lib/features/splash/presentation/cubit/splash_state.dart

```diff
@@ -0,0 +1,14 @@
+part of 'splash_cubit.dart';
+
+abstract class UserState {
+  const UserState();
+}
```

Comment:
why the file is splash state but the class ios user state?

Reply:
Fixed